test/formulas/TESTS.json

```diff
@@ -175468,5 +175468,51 @@
           2
       ]
     ]
+  },
+  "SORT sorts on multiple columns": {
+    "expectedValue": true,
+    "formula": "AND(SORT(A1:C5, 1, 1, FALSE(), 2, 1, 3, 1)=D1:F5)",
```

Comment:
Can we just test sort without AND?

Reply:
it seems like, from experimenting a bit,`expectedValue` has to be a single cell value, so i'm using the AND here to check all of them at once.